_posts/2023-07-17-13.0.0-release.md

```diff
@@ -0,0 +1,324 @@
+
+#### Arrow
+
+* It is now possible to build the Arrow Go lib using tinygo for building smaller WASM binaries ([GH-32832](https://github.com/apache/arrow/issues/32832))
+* `Fields` method for Schema and StructType now returns a copy of the slice to ensure immutability ([GH-35306](https://github.com/apache/arrow/issues/35306) and [GH-35866](https://github.com/apache/arrow/issues/35866))
+* `array.ApproxEqual` for Maps now allows entries for a given element to be presented in any order ([GH-35828](https://github.com/apache/arrow/issues/35828))
+* Fix issues with decimal256 arrays ([GH-35911](https://github.com/apache/arrow/issues/35911), [GH-35965](https://github.com/apache/arrow/issues/35965), and [GH-35975](https://github.com/apache/arrow/issues/35975))
+* StructType now allows duplicate field names correctly ([GH-36014](https://github.com/apache/arrow/issues/36014))
+
+#### Flight
+
+* Fix crash in client middleware ([GH-35240](https://github.com/apache/arrow/issues/35240))
+
+#### Parquet
+
+* Various memory leaks addressed in pqarrow package ([GH-35015](https://github.com/apache/arrow/issues/35015))
+* Fixed panic for `ListOf(types)` if null ([GH-35684](https://github.com/apache/arrow/issues/35684))
+
+
+## Java notes
+
+The JNI bindings for Arrow Dataset now support execute [Substrait](https://substrait.io/) plans via the [Acero](https://arrow.apache.org/docs/dev/cpp/streaming_execution.html) query engine. ([#34223](https://github.com/apache/arrow/issues/34223))
+
+Arrow packages that depend on Netty (most notably, `arrow-memory-netty`, but also Arrow Flight) now require Netty 4.1.94.Final at a minimum. ([#36209](https://github.com/apache/arrow/issues/36209))
+
+`VectorSchemaRoot#slice` now always makes a copy, including when the slice covers all rows (previously it did not make a copy in this case). This is a potentially-breaking change if your application depended on the old behavior. ([#35275](https://github.com/apache/arrow/issues/35275))
+
+Debug info for allocations is no longer automatically enabled when assertions are enabled (e.g. when running unit tests). Instead, support must be explicitly enabled. This is not quite a breaking change, but may be surprising if you are used to using this information while debugging tests. However, performance should be greatly improved while running tests. ([#34338](https://github.com/apache/arrow/issues/34338))
+
+Support for the upcoming Java 21 was added, though we do not yet test this in CI ([#5053](https://github.com/apache/arrow/issues/35053)). The JNI bindings for Arrow Dataset now expose JSON support ([#36421](https://github.com/apache/arrow/issues/36421)). Dictionary replacement is now supported when writing the IPC stream format ([#18547](https://github.com/apache/arrow/issues/18547)).
+
+## JavaScript notes
+
+## Python notes
+
+Compatibility notes:
+
+* The default format version for Parquet has been bumped from 2.4 to 2.6 [GH-35746](https://github.com/apache/arrow/issues/35746)
```

Comment:
```suggestion
* The default format version for Parquet has been bumped from 2.4 to 2.6 [GH-35746](https://github.com/apache/arrow/issues/35746). In practice, this means that nanosecond timestamps now preserve its resolution instead of being converted to microseconds.
```

Reply:
Perhaps "now preserve their resolution"?